apps/meteor/ee/server/models/raw/ServiceLevelAgreements.ts

```diff
@@ -9,6 +9,13 @@ export class ServiceLevelAgreements extends BaseRaw<IOmnichannelServiceLevelAgre
 		super(db, 'omnichannel_service_level_agreements');
 	}
 
+	protected modelIndexes(): IndexDescription[] {
+		return [
+			{ key: { name: 1 }, unique: true },
+			{ key: { dueTimeInMinutes: 1 }, unique: true },
```

Comment:
I'm assuming since this SLA feature is going to be deprecated as is, we're not creating any special error handlers for these indexes :eyes:

Reply:
Which special error handlers would you create? 👀